AppKit/CPWindow/_CPWindowView.j

```diff
@@ -353,7 +353,10 @@ _CPWindowViewResizeSlop = 3;
     if ([theWindow isFullPlatformWindow] ||
         !(_styleMask & CPResizableWindowMask) ||
         (CPWindowResizeStyle !== CPWindowResizeStyleModern))
+    {
+        [[CPCursor arrowCursor] set];
```

Comment:
If you implement cursorUpdate: correctly in CPResponder, this should not be necessary, and in fact this may lead to incorrect behavior, because if a tracking area owner implements cursorUpdate: and does not set the cursor and does not call super, the cursor does not change.